frontend/src/components/common/GenericModal.tsx

```diff
@@ -1,5 +1,12 @@
 import React, { useState, useEffect } from 'react';
 import { Modal, Button, Container } from 'react-bootstrap';
+import classNames from 'classnames';
+
+export enum ModalSize {
+  EXTRA_LARGE = 'modal-xl',
```

Comment:
Might want to save your fingers and rename `XLARGE`

Reply:
fair, I'll get this in a future PR.